_posts/elements/2015-07-01-grid.md

```diff
@@ -0,0 +1,710 @@
+                <span class="example-label">col-2</span>
+            </div>
+
+            <div class="col col-2">
+                <span class="example-label">col-2</span>
+            </div>
+
+            <div class="col col-2">
+                <span class="example-label">col-2</span>
+            </div>
+
+            <div class="col col-2">
+                <span class="example-label">col-2</span>
+            </div>
+
+            <div class="col col-2">
+                <span class="example-label">col-2</span>
+            </div>
+        </div>
+
+        <div class="row">
+            <div class="col col-3">
+                <span class="example-label">col-3</span>
+            </div>
+
+            <div class="col col-3">
+                <span class="example-label">col-3</span>
+            </div>
+
+            <div class="col col-3">
+                <span class="example-label">col-3</span>
+            </div>
+
+            <div class="col col-3">
+                <span class="example-label">col-3</span>
+            </div>
+        </div>
+
+        <div class="row">
+            <div class="col col-4">
+                <span class="example-label">col-4</span>
+            </div>
+
+            <div class="col col-4">
+                <span class="example-label">col-4</span>
+            </div>
+
+            <div class="col col-4">
+                <span class="example-label">col-4</span>
+            </div>
+        </div>
+
+        <div class="row">
+            <div class="col col-12">
+                <span class="example-label">col-12</span>
+            </div>
+        </div>
+
+        <div class="row">
+            <div class="col col-2 pre-5 post-5">
+                <span class="example-label">col-2</span>
+                <span class="example-label">pre-5</span>
+                <span class="example-label">post-5</span>
+            </div>
+
+            <div class="col col-4 pre-4 post-4">
+                <span class="example-label">col-4</span>
+                <span class="example-label">pre-4</span>
+                <span class="example-label">post-4</span>
+            </div>
+
+            <div class="col col-6 pre-3 post-3">
+                <span class="example-label">col-6</span>
+                <span class="example-label">pre-3</span>
+                <span class="example-label">post-3</span>
+            </div>
+
+            <div class="col col-8 pre-2 post-2">
+                <span class="example-label">col-8</span>
+                <span class="example-label">pre-2</span>
+                <span class="example-label">post-2</span>
+            </div>
+
+            <div class="col col-10 pre-1 post-1">
+                <span class="example-label">col-10</span>
+                <span class="example-label">pre-1</span>
+                <span class="example-label">post-1</span>
+            </div>
+        </div>
+    </div>
+</div>
+
+<h3 class="hd-6 example-set-hd">Utility Class-based Nested Grid</h3>
```

Comment:
I think we talked this over in person and I lean toward not allowing nesting through the DOM-based method at this time. If we find there is pent-up demand (businessbusinessbusiness!) for them, we can tackle it once we have a better handle on the needs, and whether it is more DOM-based or layout based.